docs/content/drafts/DropdownMenu2.mdx

```diff
@@ -0,0 +1,308 @@
+---
+component_id: dropdown_menu
+title: DropdownMenu v2
+status: Alpha
+source: https://github.com/primer/react/tree/main/src/DropdownMenu2.tsx
+storybook: '/react/storybook?path=/story/composite-components-dropdownmenu2'
+description: Use DropdownMenu to select a single option from a list of menu options.
+---
+
+import {Box, Text, Avatar} from '@primer/react'
+import {DropdownMenu, ActionList} from '@primer/react/drafts'
+import {Props} from '../../src/props'
+import State from '../../components/State'
+import {CalendarIcon, IterationsIcon, NumberIcon, SingleSelectIcon, TypographyIcon} from '@primer/octicons-react'
+
+<br />
+
+<State default={1}>
```

Comment:
This example appears at the top of docs, but with more recent updates we've been moving them under examples heading and keeping import statements at the top. Can we do the same here for consistency?

Reply:
Yep!

I added an example on the top for ActionList, ActionMenu as well to quickly see what this component is. Meanwhile, the examples slowly build up starting from an easy example to more complex ones.

Reply:
Looks quite nice actually, almost like a preview / hero.

👍 .. we'll need to comm this to others so we can consider doing same elsewhere.